client/src/components/questions/AnswerModal.jsx

```diff
@@ -0,0 +1,60 @@
+/* eslint-disable react/prop-types */
+import React from 'react';
+import styled, { createGlobalStyle } from 'styled-components';
+
+const StyledBackground = styled.div`
```

Comment:
Consider creating a file for your styledComponents that you can then just import the required components into the file? This will make it easier for us to compare the styles towards the end of the project!

Reply:
good idea, I will look into that.